### PR TITLE
IRISHUB-973:edit docs

### PR DESCRIPTION
**Pull request by kidinamoto01**

### CODE REVIEW

File: docs/zh/features/governance.md [hunk 1]
```diff
@@ -2,9 +2,10 @@
 
 ## 基本功能描述
 
-1. 文本提议的链上治理
```
Comment (wukongcheng) [inline docs/zh/features/governance.md]: 文本提议 在这个版本还是有的

File: docs/zh/features/service.md [hunk 2]
```diff
@@ -208,4 +210,4 @@ iriscli service refund-deposit --chain-id=service-test  --from=x --fee=0.004iris
 
 * IDL文件参照
 
-    [test.proto](https://github.com/irisnet/irishub/blob/master/docs/features/test.proto)
+    [test.proto](./test.proto)
```
Comment (wukongcheng) [inline docs/zh/features/service.md]: Use the URI

File: docs/zh/features/upgrade.md [hunk 3]
```diff
@@ -7,15 +7,15 @@
 ## 交互流程
 
 ### 软件升级提议治理流程
-1. 用户提交升级软件的提议
+1. 用户提交升级软件的提议并且进过投票使该提议通过
```
Comment (wukongcheng) [inline docs/zh/features/upgrade.md]: 进过 ？

File: docs/zh/cli-client/upgrade/query-signals.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+打印帮助信息:
+
+```
+iriscli upgrade query-signals --help
+```
+## 标志
+
+| 名称, 速记       | 默认值                     | 描述                                                        | 必需     |
+| --------------- | -------------------------- | ----------------------------------------------------------------- | -------- |
+
```
Comment (wukongcheng) [inline docs/zh/cli-client/upgrade/query-signals.md]: flags is empty ?

File: docs/features/governance.md [hunk 5]
```diff
@@ -2,9 +2,10 @@
 
 ## Basic Function Description
 
-1. On-chain governance proposals on text
-2. On-chain governance proposals on parameter change
-3. On-chain governance proposals on software upgrade 
+1. On-chain governance proposals on parameter change
```
Comment (wukongcheng) [inline docs/features/governance.md]: Still have the text proposal

File: docs/features/basic-concepts/gov-params.md [hunk 6]
```diff
@@ -1,18 +1,27 @@
...
+   * Voting period as `voting_period` in genesis file
+* In `TallyingProcedure`  step of governance procedure, the following parameters are up to on-chain governance:
+   * Threshold as `threshold` in genesis file to pass a proposal 
+   * Veto percentage as `veto`in genesis file to stop a proposal 
+   * Participation percentage as `participation` in genesis file to make the results legitimate
 
 Details in [gov](../governance.md)
 
-## Service Module
+## Parameters inService Module
```
Comment (wukongcheng) [inline docs/features/basic-concepts/gov-params.md]: inService ?

File: docs/features/basic-concepts/genesis-file.md [hunk 7]
```diff
@@ -50,4 +53,6 @@ The Genesis file is the basis for the entire network initialization，which cont
   
 ## Gentxs
 
-Gentxs contains the transaction set of creating validators in genesis block. 
\ No newline at end of file
+Gentxs contains the transaction set of creating validators in genesis block. 
+The IRIS SDK provides robust tools for bootstrapping the identities that will start chain via the gen-tx process. gen-tx or a Genesis Transaction are cryptographically signed transactions that are executed during chain initialization that generate a starting set of validators.
```
Comment (wukongcheng) [inline docs/features/basic-concepts/genesis-file.md]: we don't have the `IRIS SDK`

File: docs/cli-client/upgrade/query-signals.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+Print help messages:
+
+```
+iriscli upgrade query-signals --help
+```
+## Flags
+
+| Name, shorthand | Default                    | Description                                                       | Required |
+| --------------- | -------------------------- | ----------------------------------------------------------------- | -------- |
```
Comment (wukongcheng) [inline docs/cli-client/upgrade/query-signals.md]: flags is empty?

File: docs/zh/cli-client/upgrade/query-signals.md (hunk 4 above)
Comment (kidinamoto01) [reply docs/zh/cli-client/upgrade/query-signals.md]: fixed

File: docs/features/basic-concepts/gov-params.md (hunk 6 above)
Comment (kidinamoto01) [reply docs/features/basic-concepts/gov-params.md]: fixed

File: docs/features/basic-concepts/genesis-file.md (hunk 7 above)
Comment (kidinamoto01) [reply docs/features/basic-concepts/genesis-file.md]: fixed

File: docs/zh/features/basic-concepts/genesis-file.md [hunk 9]
```diff
@@ -50,4 +51,5 @@ Genesis文件是整个网络初始化的基础。它包含了创建创世区块
   
 ## Gentxs
 
-Gentxs中包含了创世区块中创建validators的交易集合。
+Gentxs中包含了创世区块中创建validators的交易集合。Cosmos SDK通过一个 gen-tx可以有效的对genesis文件的产生进行管控。 gen-tx即Genesis Transaction 是经过签名的交易数据，在区块链启动的过程中，这些交易将被执行，然后确定初始验证人集合。
```
[the remark this answers is not in the document]
Comment (kidinamoto01) [reply docs/zh/features/basic-concepts/genesis-file.md]: fixed

File: docs/zh/features/governance.md (hunk 1 above)
Comment (kidinamoto01) [reply docs/zh/features/governance.md]: fixed

File: docs/zh/features/upgrade.md (hunk 3 above)
Comment (kidinamoto01) [reply docs/zh/features/upgrade.md]: fixed

File: docs/features/governance.md (hunk 5 above)
Comment (kidinamoto01) [reply docs/features/governance.md]: fixed

**Comment by wukongcheng** [comment]
Why delete query-signals.md?

**Pull request merged**